Makefile

```diff
@@ -66,13 +68,19 @@ e2e-test:
 
 .PHONY: e2e-bootstrap
 e2e-bootstrap: install-helm
-	# Only build and push the image if it does not exist in the registry
 	docker pull $(IMAGE_TAG) || make azuredisk-container push
+ifdef TEST_WINDOWS
+	helm install azuredisk-csi-driver charts/latest/azuredisk-csi-driver --namespace kube-system --wait --timeout=15m -v=5 --debug \
+		--set image.azuredisk.repository=$(REGISTRY)/$(IMAGE_NAME) \
+		--set image.azuredisk.tag=$(IMAGE_VERSION) \
+		--set controller.replicas=1 \
```

Comment:
remove this `--set controller.replicas=1` since we already have 3 master node now, let's use the default value(2)